src/main/java/br/com/harbitech/school/activity/Activity.java

```diff
@@ -14,7 +16,7 @@
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String title;
```

Comment:
Poderia marcar aqui com a validação `@NotBlank`:

```suggestion
   @NotBlank
    private String title;
```